spec/lib/whitehall_importer/create_image_revision_spec.rb

```diff
@@ -86,6 +86,19 @@
       end
     end
 
+    context "Image has exif data" do
+      let(:whitehall_image) do
+        build(:whitehall_export_image, fixture_file: "960x640-rotated.jpg")
+      end
+
+      it "should strip the exif data from the image" do
+        revision = described_class.call(document_import, whitehall_image, ["valid-image.jpg"])
+
+        image = MiniMagick::Image.open(revision.image.image_revisions.last.blob_revision.blob)
```

Comment:
I'd have thought you could do this with just `revision.blob`

Reply:
You can indeed, that looks much cleaner!